### PR TITLE
Rehash of #150

### PR DESCRIPTION
**Pull request by githubuser4141**
## About The Pull Request

Re-do of #150 since I messed up and deleted it by accident. Should be modularized (?) now?

## Why It's Good For The Game

Read [#150](https://github.com/Jackriip/Voidcrew-LRP/pull/150) 

## Changelog
:cl:
tweak: Mechs no longer require default access, you can print ID robotics ID add-ons from RnD, Twinkleshine captains can access their office and armory now
/:cl:

### CODE REVIEW

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -571,16 +571,6 @@ update_label()
 	access = list(ACCESS_SYNDICATE)
 	uses_overlays = FALSE
 
-/obj/item/card/id/syndicate_command/captain_id
```
Comment (SuperNovaa41) [inline code/game/objects/items/cards_ids.dm:574]: Did you mean to delete this?

**Review by SuperNovaa41 — APPROVED** [review]
Looks good!

File: code/game/objects/items/cards_ids.dm (hunk 1 above)
Comment (githubuser4141) [reply code/game/objects/items/cards_ids.dm:574]: For some reason there's two syndi captain IDs with the same path, I deleted the one with only one type of access so twinkleshine caps wouldn't spawn with it and not be able to access anything

**Comment by Jackriip** [comment]
test merging

**Comment by SuperNovaa41** [comment]
Full merging

**Pull request merged**